notebooks/WFC3/persistence/wfc3_ir_persistence.ipynb

```diff
@@ -58,9 +58,9 @@
     "\n",
```

Comment:
<p>Thanks for the updates on this cell. I suggest adding exception handling to catch errors that may occur during execution.</p><pre class="ql-syntax" spellcheck="false">url = 'https://archive.stsci.edu/pub/wfc3_persist/13459/Visit29/13459.Visit29.tar.gz'
filename = '13459.Visit29.tar.gz'

try:
&nbsp;&nbsp;with urllib.request.urlopen(url) as response:
&nbsp;&nbsp;&nbsp;&nbsp;with open(filename,'wb') as out_file:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;out_file.write(response.read())
&nbsp;&nbsp;print('Download complete.')

&nbsp;&nbsp;# Extract files:
&nbsp;&nbsp;!tar -zxvf {filename}

&nbsp;&nbsp;# Remove the files after extraction
&nbsp;&nbsp;os.remove('13459.Visit29.tar.gz')
except Exception as e:
&nbsp;&nbsp;print(f"An error occured: {e}")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>